elasticsearch/examples/networkpolicy/Makefile

```diff
@@ -0,0 +1,15 @@
+default: test
+include ../../../helpers/examples.mk
+
+RELEASE := helm-es-networkpolicy
+
+install:
+	helm upgrade --wait --timeout=600 --install $(RELEASE) --values ./values.yaml ../../ ; \
```

Comment:
timeout option now requires `s` for second with Helm 3

Reply:
Updated!